File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //set restrictions
    int n;
    do
    {
        n = get_int("Size: /n");
    }
    while (n < 1 || n > 8);
    //Row iternation
    for (int i = 1; i <= n; i++)
    {
        //Left side
        for (int j = 1; j <= n - i; j++)
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf(" ");
        //Right side
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: mario.py
from cs50 import get_int



while True:
    n = get_int("Enter int between 1 and 8: ")                 #prompt user for number
    if 0 < n < 9:
        break

for i in range(n):                                              #for each row n rows
    for j in range(n-1-i):
        print(" ", end="")
    for j in range(i+1):                                        #for number of times as row number, print j times followed by 2 spaces
        print("#", end="")
    print("  ", end="")
    for j in range(i+1):                                         #for number of times as row number, print j times
        print("#", end="")
    print()
